pkg/v1/tkg/test/tkgctl/tkgs/tkgs_workload_cluster_test.go

```diff
@@ -131,38 +131,112 @@ var _ = Describe("TKGS - Create workload cluster use cases", func() {
+
+				createClusterClassBasedCluster(tkgctlClient, true, clusterName, namespace)
+				/* Note: Currently not able to use CheckClusterCB function becuase the management cluster details required are not available
+
+				var clusterResources []shared.ClusterResource
+
+				By(fmt.Sprintf("Get k8s client for management cluster %q", clusterName))
+				mngkubeConfigFileName := clusterName + ".kubeconfig"
+				mngTempFilePath := filepath.Join(os.TempDir(), mngkubeConfigFileName)
+				err = tkgctlClient.GetCredentials(tkgctl.GetWorkloadClusterCredentialsOptions{
+					ClusterName: clusterName,
+					Namespace:   namespace,
+					ExportFile:  mngTempFilePath,
+				})
+				Expect(err).To(BeNil())
+
+				By(fmt.Sprintf("Get k8s client for management cluster %q", clusterName))
+				mngClient, mngDynamicClient, mngAggregatedAPIResourcesClient, mngDiscoveryClient, err := shared.GetClients(context.Background(), mngTempFilePath)
+				Expect(err).NotTo(HaveOccurred())
+
+				By(fmt.Sprintf("Generating credentials for workload cluster %q", e2eConfig.WorkloadClusterOptions.ClusterName))
+				wlcKubeConfigFileName := e2eConfig.WorkloadClusterOptions.ClusterName + ".kubeconfig"
+				wlcTempFilePath := filepath.Join(os.TempDir(), wlcKubeConfigFileName)
+				err = tkgctlClient.GetCredentials(tkgctl.GetWorkloadClusterCredentialsOptions{
+					ClusterName: clusterName,
+					Namespace:   namespace,
+					ExportFile:  wlcTempFilePath,
+				})
+				Expect(err).To(BeNil())
+
+				By(fmt.Sprintf("Get k8s client for workload cluster %q", clusterName))
+				wlcClient, _, _, _, err := shared.GetClients(context.Background(), wlcTempFilePath)
+				Expect(err).NotTo(HaveOccurred())
+
+				By(fmt.Sprintf("Verify addon packages on management cluster %q matches clusterBootstrap info on management cluster %q", clusterName, clusterName))
+				err = shared.CheckClusterCB(context.Background(), mngClient, wlcClient, clusterName, namespace, "", "", e2eConfig.InfrastructureName, true)
+				Expect(err).To(BeNil())
+
+				By(fmt.Sprintf("Verify addon packages on workload cluster %q matches clusterBootstrap info on management cluster %q", e2eConfig.WorkloadClusterOptions.ClusterName, clusterName))
+				err = shared.CheckClusterCB(context.Background(), mngClient, wlcClient, clusterName, namespace, clusterName, namespace, e2eConfig.InfrastructureName, false)
+				Expect(err).To(BeNil())
+
+				By(fmt.Sprintf("Get management cluster resources created by addons-manager for workload cluster %q on management cluster %q", e2eConfig.WorkloadClusterOptions.ClusterName, clusterName))
+				clusterResources, err = shared.GetManagementClusterResources(context.Background(), mngClient, mngDynamicClient, mngAggregatedAPIResourcesClient, mngDiscoveryClient, namespace, clusterName, e2eConfig.InfrastructureName)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(clusterResources).ToNot(BeEmpty())
+				*/
```

Comment:
All of these is commented out. 
If it will not be implemented in this patch, best to remove it.